code/controllers/subsystem/lisp.dm

```diff
@@ -0,0 +1,16 @@
+SUBSYSTEM_DEF(lisp)
+	name = "LISP"
+	offline_implications = "Silicon units cannot recover integrity. Shuttle call recommended."
+	runlevels = RUNLEVEL_GAME
+	wait = 3 SECONDS
+
+/datum/controller/subsystem/lisp/Initialize()
+	return ..()
+
+/datum/controller/subsystem/lisp/fire()
+	// allow the AIs to recover integrity
+	for(var/mob/living/silicon/ai/A in GLOB.alive_mob_list)
+		A.adjust_integrity(1)
+	// allow the borgs to recover integrity
+	for(var/mob/living/silicon/robot/B in GLOB.alive_mob_list)
+		B.adjust_integrity(1)
```

Comment:
I think looping and filtering through *all* alive mobs every few seconds like this every is inefficient. IMO you should remove this subsystem and instead call the `adjust_integrity()` proc in `/silicon/Life()`. Maybe override it for `/drone`'s if you don't want them to be affected.